desktop/app/store-helper.js

```diff
@@ -20,7 +20,10 @@ export default function (mainWindow) {
     subscribeStoreSubscribers.push({sender, selector})
 
     try {
-      sender.send('stateChange', selector ? selector(store) : store)
+      const newStore = selector ? selector(store) : store
+      if (newStore) {
```

Comment:
don't send updates if the store is just not there, likely due to a timing problem